packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -173,13 +230,15 @@ const extractTemplateLiteral = (node: t.TemplateLiteral, meta: Metadata): CSSOut
       nextQuasis.value.raw = after.css; // Removes any suffixes from the next quasis.
       variables.push({ name: variableName, expression });
 
-      return before.css + `var(${variableName})`;
+      return acc + before.css + `var(${variableName})`;
```

Comment:
interestingly this was a bug fix.... how did this not fail before!

Reply:
Fug!